tests/bdd/common/deployer.py

```diff
@@ -1,34 +1,82 @@
 import os
 import subprocess
 
+from dataclasses import dataclass
+
+
+@dataclass
+class StartOptions:
+    mayastors: int = 1
+    wait: str = "10s"
+    csi: bool = False
+    reconcile_period: str = ""
+    cache_period: str = ""
+    mayastor_env: str = ""
+    agents_env: str = ""
+    jaeger: bool = True
+    extra_args: [str] = ()
+
+    def args(self):
+        args = [
+            "--mayastors",
+            str(self.mayastors),
+            "--wait-timeout",
+            self.wait,
+        ]
+        if self.csi:
+            args.append("--csi")
+        if self.jaeger:
+            args.append("--jaeger")
+        if len(self.reconcile_period) > 0:
+            args.append(f"--reconcile-period={self.reconcile_period}")
+            args.append(f"--reconcile-idle-period={self.reconcile_period}")
+        if len(self.cache_period) > 0:
+            args.append(f"--cache-period={self.cache_period}")
+        if len(self.mayastor_env) > 0:
+            args.append(f"--mayastor-env={self.mayastor_env}")
+        if len(self.agents_env) > 0:
+            args.append(f"--agents-env={self.agents_env}")
+        if len(self.extra_args) > 0:
+            args.append(self.extra_args)
+        return args
+
 
 class Deployer(object):
-    # Start containers with the default arguments.
+    # Start containers with the specified function
```

Comment:
Should "function" be "default values" here?

Reply:
doh yeah or with the provided values